src/adhocracy_core/adhocracy_core/testing.py

```diff
@@ -665,6 +673,17 @@ def newest_activation_path(app):
 
 
 @fixture(scope='class')
+def newest_reset_path(app):
+    """Return the newest reset password path generated by app."""
+    from urllib.request import unquote
+    mailer = app.registry.messenger._get_mailer()
+    last_message_body = mailer.outbox[-1].body
```

Comment:
Not sure how this plays with the change to the mailer in the .ini files that I did when I wrote the Protractor. I changed the mailer to receive the mails on the disk.